docs/getting-started/developers/index.md

```diff
@@ -118,6 +118,10 @@ For more examples how to configure other building tools, please refer to the [ex
 
 #### Add uno.css to your main entry
 
+Depending on the build tool you have chosen this step can look different. The example below is applicable if you build unocss together with Vite. If
```

Comment:
UnoCSS? 😇

Reply:
ah, took me a while to find it 👁️ I fix, tnx